src/DotVVM.Framework/Compilation/Binding/ExpressionHelper.cs

```diff
@@ -85,6 +85,42 @@ static Expression GetDotvvmPropertyMember(Expression target, string name)
             );
         }
 
+        public static Expression SetMember(Expression node, Expression value)
+        {
+            if ((node.NodeType == ExpressionType.MemberAccess
+                && node is MemberExpression member
+                && member.Member is PropertyInfo property
+                && property.CanWrite)
+                || node.NodeType == ExpressionType.Parameter
+                || node.NodeType == ExpressionType.Index)
+            {
+                return Expression.Assign(node, Expression.Convert(value, node.Type));
+            }
+
+            var current = node;
+            while (current.NodeType == ExpressionType.Convert
+                && current is UnaryExpression unary)
+            {
+                current = unary.Operand;
+            }
+
+            if (current.NodeType == ExpressionType.Call
+                && current is MethodCallExpression call
+                && call.Method.DeclaringType == typeof(DotvvmBindableObject)
+                && call.Method.Name == nameof(DotvvmBindableObject.GetValue)
+                && call.Arguments.Count == 2
```

Comment:
Could you please add an unit test for this case, so we at don't break it by adding an optional parameter to the GetValue method.

Reply:
I thought of a unit test of this method, not a check that that overload still exists. You actually have a bug in here, the signature of `GetValue` is `GetValue(DotvvmProperty, bool)` not `GetValue(DotvvmProperty, object)`.

Reply:
I have added few tests for that and fixed this problem.